model/src/main/proto/topic.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+// Represents the play state of a single chapter.
+enum ChapterPlayState {
+  // The completion status is unknown.
+  COMPLETION_STATUS_UNSPECIFIED = 0;
+
+  // The chapter has not yet been started, but can be started by the player.
+  NOT_STARTED = 1;
+
+  // The chapter has not yet been started, and cannot be started since the player is missing prerequisites.
+  NOT_PLAYABLE_MISSING_PREREQUISITES = 2;
+
+  // The chapter has been completed by the player.
+  COMPLETED = 3;
+}
+
+// Represents the progress a player has made for a story.
+message StoryProgress {
+  // Represents the progress a learner has made on a single chapter. These chapters are kept in the same order they
+  // should be completed within the story.
```

Comment:
Nit change: add space before `should be` as per my comment on #175

Reply:
See my comment in #175, I actually don't think we want the double space here since it isn't a TODO comment.